mongodbatlas/clusters.go

```diff
@@ -153,6 +153,7 @@ type Cluster struct {
 	MongoURIUpdated          string                   `json:"mongoURIUpdated,omitempty"`
 	MongoURIWithOptions      string                   `json:"mongoURIWithOptions,omitempty"`
 	Name                     string                   `json:"name,omitempty"`
+	CreateDate               string                   `json:"createDate,omitempty"`
```

Comment:
[q] are these and `Links` serverless only? if so can we add a godoc saying so?

Reply:
yes, they are. sure

Reply:
I would also be in favour of maybe duplicating the create params with what you only need for create, it could be confusing for users to understand what params are dedicated clusters only and which ones are not

Reply:
no, sorry. They are also used with normal clusters

Reply:
`Links` is in the example of response for GET CLUSTER https://docs.atlas.mongodb.com/reference/api/clusters-get-one/

<img width="716" alt="Screenshot 2021-07-23 at 14 48 39" src="https://user-images.githubusercontent.com/5663078/126783742-96a1ecd2-2450-4823-9bd4-c004f21f8083.png">